src/Microsoft.CodeAnalysis.Analyzers/Test/MetaAnalyzers/StartActionWithNoRegisteredActionsRuleTests.cs

```diff
@@ -194,6 +206,117 @@ private static void AnalyzeCodeBlockStart(CodeBlockStartAnalysisContext<SyntaxKi
         }
 
         [Fact]
+        public void CSharp_NoDiagnosticCases_3()
```

Comment:
More descriptive names for test cases. Maybe `CSharp_NoDiagnostic_OperationAnalyzerRegistrationIncludesOperationKind`.